src/mixins/amalgamation-mixin.ts

```diff
@@ -55,42 +55,48 @@ export default class AmalgamationMixin extends Vue {
   //   }
   // ]
 
+  /**
+   * Get the amalgamating business information, mailing address, and email if in LEAR.
+   * Otherwise, return error.
+   */
+  async fetchTingInfo (item: any): Promise<AmalgamatingBusinessIF> {
+    let business = null
+    const data = await Promise.all([
+      LegalServices.fetchBusinessInfo(item.identifier),
+      AuthServices.fetchAuthInfo(item.identifier),
+      LegalServices.fetchAddresses(item.identifier)
```

Comment:
@severinbeauvais Sev, once you merge your changes, I'll change this function here to whatever you decide upon in your PR.

Reply:
Thanks. I hope to have something very soon. Just testing atm.

Reply:
Shall we resolve this conversation now?

Reply:
Sure thing! This is super outdated now anyway since a lot has changed.